src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -49,6 +49,14 @@ unsafe static int Execute(int argc, char** argv, NativeBootstrapperContext* cont
             bootstrapperContext.TargetFramework = new FrameworkName(FrameworkNames.LongNames.DnxCore, new Version(5, 0));
             bootstrapperContext.HandleExceptions = context->HandleExceptions;
 
+            Logger.TraceInformation($"Operating System: {bootstrapperContext.OperatingSystem}");
```

Comment:
We shouldn't log this here if application host logs it. We should do it in Dnx.Host so it's consistent.